media/css/firefox/whatsnew/whatsnew-61.scss

```diff
@@ -0,0 +1,404 @@
+    }
+
+    @media #{$mq-tablet} {
+        margin: 0 auto;
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Main page content header general styles
+
+.content-header {
+    padding: 20px;
+    position: relative;
+    text-align: center;
+    margin-bottom: 20px;
+
+    h2 {
+        @include font-size-level2;
+        margin-bottom: .5em;
+    }
+
+    h3 {
+        @include font-size-level5;
+        margin-top: 40px;
+    }
+
+    .tagline {
+        @include font-size-level4;
+        line-height: 1.5;
+        margin: 0 auto 20px;
+
+        li {
+            margin-bottom: .5em;
+        }
+    }
+
+    p {
+        @include font-size-level6;
+    }
+
+    .more {
+        @include font-size-level5;
+
+        a:link,
+        a:visited {
+            color: #fff;
+        }
+    }
+
+    @media #{$mq-tablet} {
+        padding: 0;
+        text-align: left;
+
+        .tagline {
+            margin: 0 0 30px;
+        }
+
+        ul.tagline {
+            list-style: disc;
+            padding-left: 1em;
+        }
+    }
+}
+
+html[dir="rtl"] .content-header {
+    @media #{$mq-tablet} {
+        text-align: right;
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Firefox accounts
+
+#fxa-iframe-config {
+    display: none;
+}
+
+.show-fxa {
+    .fxa-container {
+        display: block;
+    }
+
+    #fxa {
+        border: none;
+        min-height: 220px;
+        position: relative;
+        width: 300px;
+    }
+
+    #fxa-iframe-config {
+        background: #fff;
+        display: block;
+        margin: 20px auto;
+        padding: 30px 0;
+        width: 300px;
+    }
+
+    @media #{$mq-tablet} {
+        // sets width for logo and content containers
+        .content-container {
+            width: $width-tablet - 40px;
+        }
+
+        .fxa-container {
+            display: grid;
+            grid-column-gap: 20px;
+            grid-template-columns: repeat(2, 1fr);
+        }
+
+        #fxa {
+            width: auto;
+        }
+
+        #fxa-iframe-config {
+            margin: 0;
+            padding: 30px 20px;
+            width: auto;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .content-container {
+            width: $width-desktop - 40px;
+        }
+
+        .fxa-container {
+            grid-column-gap: 60px;
+            grid-template-columns: 1fr 350px;
+        }
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Firefox mobile - 2 possible states:
+// 1. Send to Device widget
+// 2. QR code (if widget is not available)
+// Also the default state if JS isn't available.
+
+.has-send-to-device.show-fx-mobile {
+    .fx-mobile-container {
+        display: block;
+    }
+
+    @media #{$mq-tablet} {
+        .content-container {
+            width: 600px;
+        }
+    }
+}
+
+.no-send-to-device.show-fx-mobile {
+    @media #{$mq-tablet} {
```

Comment:
I think we need a `display: block` (or equivalent) prior to the media query here, as there's no content visible when `< 760px` for this set of conditions.